app/services/messages.js

```diff
@@ -0,0 +1,30 @@
+import Ember from "ember";
+const { getOwner } = Ember;
+
+export default Ember.Service.extend({
+  logger: Ember.inject.service(),
+  session: Ember.inject.service(),
+
+  markRead: function (message) {
```

Comment:
as discussed on slack, it would be good to look at a way to push a list of messages to mark read rather than lots of inidividual 'mark_read' calls.

Reply:
yep, right but @steveyken, for that we'll need to make changes to all the 4 apps accordingly. As currently on `donor` and `admin` also the messages are marked `read` in similar fashion.
Also the api would also require changes, as currently it is designed to handle the marking of messages to `read` for a single instance of message.

Let me know should I go ahead with the changes? 

cc - @namrataukirde

Reply:
no need to change now. let's come back to it.